src/Controller/SubscriptionController.php

```diff
@@ -86,7 +86,7 @@ public function subscribe($entity_type_id, EntityInterface $group, OgMembershipT
       throw new AccessDeniedHttpException();
     }
 
-    $user = $this->entityTypeManager()->getStorage('user')->load($this->currentUser()->id());
+    $user = $this->currentUser();
```

Comment:
Exception: TypeError: Argument 2 passed to Drupal\og\MembershipManager::createMembership() must implement interface Drupal\user\UserInterface, instance of Drupal\Core\Session\AccountProxy given, called in /home/travis/build/Gizra/og/src/Og.php on line 190

Seems like this was the reason the user was loaded as such:

```suggestion
    $user = $this->entityTypeManager()->getStorage('user')->load($this->currentUser()->id());
```